Integrations/GoogleCloudStorage/GoogleCloudStorage.py

```diff
@@ -0,0 +1,566 @@
+import traceback
+
+import demistomock as demisto
```

Comment:
make sure these are the three first lines

Reply:
+not, I assume 😛